packages/lib/README.md

```diff
@@ -12,38 +12,49 @@ https://github.com/decentraland/single-sign-on <- Single Sign On webapp repo.
 npm install @dcl/single-sign-on-client
 ```
 
-2. Initialize the client as soon as the application starts or as soon as possible. It is important that initialization happens before it is used.
+2. Initialize the client
 
 ```ts
-import * as SingleSignOn from "@dcl/single-sign-on-client";
+import { SingleSignOn } from "@dcl/single-sign-on-client";
 
-const iframeSrc = "https://id.decentraland.org";
+const sso = SingleSignOn.getInstance();
 
-SingleSignOn.init(iframeSrc);
+await sso.init({
+  src: 'https://id.decentraland.org';
+})
 ```
 
-3. Use the corresponding functions to get, store and clear the identity of the user into the iframe.
+The `src` argument determines the src of the SSO iframe to be created. If no src is provided. SSO will work using the applications localstorage instead of communicating with the iframe. Given that the iframe has some limitations to which origins can interact with it (https://id.decentraland.org can only receive messages from https://\*.decentraland.org sites), for local development you should not provide a src.
```

Comment:
```suggestion
The `src` argument determines the src of the SSO iframe to be created. If no src is provided, SSO will work using the applications localstorage instead of communicating with the iframe. Given that the iframe has some limitations to which origins can interact with it (https://id.decentraland.org can only receive messages from https://\*.decentraland.org sites), for local development you should not provide a src.
```